src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> storage;
```

Comment:
The map should be public and static. Reuse it where you need. Or create an additional Dao layer to work with this map.